uber/config.py

```diff
@@ -73,9 +73,19 @@ def make_enum(self, enum_name, section, prices=False):
                 lookup[val] = desc
 
         enum_name = enum_name.upper()
-        setattr(self, enum_name + '_OPTS', opts)
-        setattr(self, enum_name + '_VARS', varnames)
-        setattr(self, enum_name + ('' if enum_name.endswith('S') else 'S'), lookup)
+        # When loading plugin's configs, we want to make sure we don't override existing enums
+        if hasattr(self, enum_name + '_OPTS'):
```

Comment:
Yup. This is the relevant line in sideboard\config.py:
```python
for config_path in get_config_files(requesting_file_path, plugin):
        # this gracefully handles nonexistent files
        temp_config.merge(configobj.ConfigObj(config_path, encoding='utf-8', interpolation=False))
```

I initially didn't want to mess with it, but it looks like we have enough complexity in what we're doing to config that it's probably better to take that approach.